newIDE/app/src/UI/SearchBar.js

```diff
@@ -243,6 +245,8 @@ const SearchBar = React.forwardRef<Props, SearchBarInterface>(
       }
     };
 
+    const filterTags = createFilterOptions();
```

Comment:
Can you use `{ limit: 200 }`?
If yes, would be good to create this outside of the component so there is one less thing to instantiate at every render.